src/translib/transformer/sw_portchannel.go

```diff
@@ -247,6 +298,32 @@ var YangToDb_lag_fallback_xfmr FieldXfmrYangToDb = func(inParams XfmrParams) (ma
     return res_map, nil
 }
 
+var DbToYang_lag_fallback_xfmr FieldXfmrDbtoYang = func(inParams XfmrParams) (map[string]interface{}, error) {
+    if log.V(3) {
+        log.Info("Entering DbToYang_lag_fallback_xfmr")
+    }
+    var err error
+    result := make(map[string]interface{})
+
+    intfType, _, ierr := getIntfTypeByName(inParams.key)
+    if ierr != nil || intfType != IntfTypePortChannel {
+        log.Error("Invalid Interface")
+        return result, err
+    }
+
+    data := (*inParams.dbDataMap)[inParams.curDb]
+
+    fallback, ok := data[PORTCHANNEL_TABLE][inParams.key].Field["fallback"]
```

Comment:
Could we use get_fallback() API ?

Reply:
get_fallback() again makes a call to DB, using (*inParams.dbDataMap) will be more efficient.